.github/workflows/build.yml

```diff
@@ -99,37 +99,53 @@ jobs:
     if: ${{ needs.changes.outputs.customer == 'true' }}
     runs-on: ubuntu-latest
     steps:
-    - name: call signadot API
-      run: |
-         echo "TODO: calling API for (customer, signadot/hotrod:${{ github.sha }})"
-      shell: bash
+    - name: curl
+      uses: wei/curl@v1.1.1
+      with:
+        args: >- 
+          -H 'Content-Type: application/json'
+          -H 'Authorization: Bearer ${{ secrets.SIGNADOT_TOKEN }}'
+          -d '{\"org\":\"signadot\",\"cluster\":\"demo\",\"namespace\":\"hotrod\",\"defaultService\":\"frontend\",\"defaultServicePort\":8080,\"images\":[{\"name\":\"signadot\/hotrod-customer\",\"newTag\":\"${{ github.sha }}\"}]}'
```

Comment:
Shouldn't the double quotes stay unescaped since they're inside single quotes? Or is the extra escaping needed because of something in the wei/curl action? Also couldn't we format this with newlines (since it's in single quotes) to make it easier to see what info is being sent?

Reply:
There was something weird I was hitting initially. it's a pain to test these, but I'll take a stab at it again as I create the github status now.